docs/wiki/Whats-new.md

```diff
@@ -135,7 +148,10 @@ Here's what's changed in Enterprise Scale/Azure Landing Zones:
 
 #### Tooling
 
+- **NEW** ALZ Portal Accelerator - adding new networking options to deploy "Bastion" and/or "Private DNS Resovler"
+  - For "Private DNS Resolver" we are deploying a default forwarding rule to `contoso.com` domain. This is a placeholder and should be updated to your own domain once ExpressRoutes or VPN sites are configured.
```

Comment:
can we not take in a parameter for the name?

Reply:
actually from reviewing `privateDnsResolver.json` i cant see we actually create a default forwarding rule at all, so this needs removing, right?